package.json

```diff
@@ -9,7 +9,7 @@
     "build": "turbo run build",
     "test": "turbo run test",
     "lint": "eslint .",
-    "prettier": "prettier --write .",
+    "prettier": "prettier --write \"**/*.{ts,tsx,cts,mts,js,jsx,cjs,mjs}\"",
```

Comment:
We should definitely include `.json` and `.md`.

Would it be more maintainable to just exclude `.mdx`?